app/src/main/java/io/intrepid/skeleton/screens/example2/Example2Fragment.java

```diff
@@ -25,7 +25,7 @@ protected int getLayoutResourceId() {
     }
 
     @Override
-    public void showText(String text) {
-        textView.setText(text);
+    public void showIpAddress(String ip) {
+        textView.setText("Your Ip address is " + ip);
```

Comment:
Despite this just being an example we should probably externalize this string - or at least put in a comment that describes why not.